.github/workflows/clean-detekt-baseline.yaml

```
@@ -5,6 +5,10 @@ on:
     # Every midnight
     - cron: "0 0 * * *"
 
+permissions:
+  contents: read
+    pull-requests: read
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)